config/initializers/location_data.rb

```diff
@@ -14,21 +14,24 @@
 
 LOCATION_POLYGON_SETTINGS = {
   regions: {
-    boundary_api: "https://ons-inspire.esriuk.com/arcgis/rest/services/Administrative_Boundaries/Regions_December_2019_Boundaries_EN_BUC/MapServer/0/query?where=1%3D1&outFields=rgn19nm,shape&outSR=4326&f=json",
-    name_key: "rgn19nm",
+    boundary_api: "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/regions/FeatureServer/0/query?where=1%3D1&outFields=GOR10NM&outSR=4326&f=json",
+    name_key: "GOR10NM",
   },
   counties: {
-    boundary_api: "https://ons-inspire.esriuk.com/arcgis/rest/services/Administrative_Boundaries/Counties_and_Unitary_Authorities_April_2019_Boundaries_UK_BUC/MapServer/0/query?where=1%3D1&outFields=ctyua19nm,shape&outSR=4326&f=json",
-    name_key: "ctyua19nm",
+    boundary_api: "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/Counties_and_Unitary_Authorities_April_2019_EW_BUC_v2/FeatureServer/0/query?where=1%3D1&outFields=ctyua19nm&outSR=4326&f=json",
+    name_key: "CTYUA19NM",
   },
   london_boroughs: {
-    boundary_api: "https://ons-inspire.esriuk.com/arcgis/rest/services/Administrative_Boundaries/Counties_and_Unitary_Authorities_April_2019_Boundaries_UK_BUC/MapServer/0/query?where=1%3D1&outFields=ctyua19nm,shape&outSR=4326&f=json",
-    name_key: "ctyua19nm",
+    boundary_api: "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/Counties_and_Unitary_Authorities_April_2019_EW_BUC_v2/FeatureServer/0/query?where=1%3D1&outFields=ctyua19nm&outSR=4326&f=json",
+    name_key: "CTYUA19NM",
   },
   cities: {
-    boundary_api: "https://ons-inspire.esriuk.com/arcgis/rest/services/Other_Boundaries/Major_Towns_and_Cities_December_2015_Boundaries_V2/MapServer/0/query?where=1%3D1&outFields=tcity15nm,shape&outSR=4326&f=json",
-    name_key: "tcity15nm",
+    boundary_api: "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/Major_Towns_and_Cities_December_2015_EW_BGG/FeatureServer/0/query?where=1%3D1&outFields=tcity15nm,shape&outSR=4326&f=json",
+    name_key: "TCITY15NM",
   },
 }.freeze
 
+DOWNCASE_COUNTIES_WITH_RING_INDICES = YAML.load_file(base_path.join("counties.yml")).map { |line|
```

Comment:
If you wanted, I think a YAML structure of `- ['County name', <index>]` might be more straightforward to parse 
```ruby
YAML.load_file(base_path.join("counties.yml")).to_h.transform_keys(&:downcase)
```